src/main/java/seedu/modtrek/logic/commands/TagCommand.java

```diff
@@ -46,8 +46,8 @@ public class TagCommand extends Command {
      */
     public static final String MESSAGE_REMOVE_TAG_SUCCESS = "Removed tag from Module: %1$s";
 
-    public static final String MESSAGE_MISSING_PREFIX = "Missing tag prefix /t.\nExample: tag cs2109s include /t CSF "
-            + "/t CSBD";
+    public static final String MESSAGE_MISSING_PREFIX = "Missing tag prefix /t.\n"
+            + "Example: tag cs2109s include /t CSF /t CSBD";
```

Comment:
I actually meant like adding an additional "\n" character.
```suggestion
    public static final String MESSAGE_MISSING_PREFIX = "Missing tag prefix /t.\n\n"
            + "Example: tag cs2109s include /t CSF /t CSBD";
```